### DISCUSSION TITLE
Clean and recognisable custom CSS variable names

### DISCUSSION THREAD

**Discussion by muzKore** [elementor/elementor#23604] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I create a custom colour and then I want to add it into custom CSS but I find it has a cryptic randomised name. It makes it hard to distinguish what the CSS variable represents.

### Describe the solution you'd like

If I add a custom colour such as Mid Grey, Elementor would incorporate it into the variable name. Eg. --{Elementor prefix}-custom-mid-grey.
So it goes lowercase and spaces become hyphens. To save duplication, if a person adds a name that is already taken, they are told that it is already in use. 
For reference, I have been told the apparently Bricks Builder and GeneratePress have it working and would be great to see in Elementor.

### Describe alternatives you've considered

Adding the CSS variables in custom CSS stylesheets but then they would not be available in Elementor.

**Comment by mgperdomo** [comment]
@muzKore We already have this request here: https://github.com/elementor/elementor/issues/12571 but despite being something very requested Elementor does not do it because it does not listen to its community, it prefers its AI and hosting service than they are not useful.

**Reply by muzKore** [reply]
Thanks for pointing that out. I search and could not locate it. Should I delete this ticket?
Yup. Could not agree more about the whole AI thing and Elementor. It's a mess. I have ChatGPT and if I need AI content, I do it there not inside my editor.
